Assertion demo in Java (#45)
grammarly/grammarly-for-developers: examples/assertion-endpoint-java/README.md

# THIS IS NOT A PRODUCTION-READY SERVER

> This server is provided to help developers experiment with Trusted Authentication.  The goal is not to provide a fully production-ready server to secure your existing Grammarly implementation.

Reviewer: ```suggestion
This server is provided to help developers experiment with trusted authentication.  The goal is not to provide a fully production-ready server to secure your existing Grammarly implementation.
```

Reviewer: We can add a link to the trusted auth docs once they are live

# Java Assertion Server

> An example Java Spring Boot server that generates valid assertions to support the Grammarly Trusted Auth system.

Reviewer: ```suggestion
The code in this directory provides an example Java Spring Boot server that generates valid assertions for trusted authentication.
```

> ## Getting started

Reviewer: ```suggestion
## Getting Started
```

You will first need to retrieve a private key in the JWK format.  You can do this from https://developer.grammarly.com/ when you "Generate new key".  Save this file in the project root with the name `private.jwk` (this has been added to `.gitignore` so you don't accidentally check in your private key).  Then run the project with the command:

    gradle bootRun

> The server should start up. You can then test it by navigating to http://localhost:8080/assertion?clientId=client_123 to test the generation of an assertion for any given client id.

Reviewer: > clientId=client_123
Is it actually clinet_123 or is that a placeholder? If it's a placeholder, let's use `YOUR_CLIENT_ID` to match the other docs

> 

Reviewer: It might be helpful to have a How it Works section (or something similar) that tells people where the code they care about is. I think it's in examples/assertion-endpoint-java/src/main/java/com/grammarly/g4d/JavaAssertionEndpointApplication.java

> 

Reviewer: ```suggestion
## Ask a Question

If you want to ask a question, share a suggestion, or chat about how you're using the Grammarly Text Editor SDK in your application, join us in the [Grammarly for Developers Discussions](https://github.com/grammarly/grammarly-for-developers/discussions).
```
